ios/FluentUI/PreviewCard/PreviewCard.swift

```diff
@@ -30,36 +30,58 @@ import SwiftUI
 
 /// View that represents the PreviewCard.
 public struct PreviewCard: View {
-
+    var isElevated: Bool
     @Environment(\.theme) var theme: FluentUIStyle
     @Environment(\.windowProvider) var windowProvider: FluentUIWindowProvider?
     /// Creates inner PreviewCard view.
     @ViewBuilder
     var innerContents: some View {
         HStack {
             Spacer()
+                /// Sets the height and width of the Preview Card.
                 .frame(minWidth: Constants.cardMinWidth, minHeight: Constants.cardMinHeight)
-        }
+                /// Sets the background border style, radius, color, and linewidth of the Preview Card.
+                .background(
+                    RoundedRectangle(cornerRadius: Constants.cardCornerRadius)
+                        .strokeBorder(Color(Color.RGBColorSpace.sRGB,
+                                            red: Constants.cardStrokeBorderColor,
+                                            green: Constants.cardStrokeBorderColor,
+                                            blue: Constants.cardStrokeBorderColor),
+                                      lineWidth: Constants.cardLineWidth)
+                /// Modifies the background shadow of PreviewCard if the card is elevated.
+                .modifyIf(isElevated, { view in view
+                        .shadow(color: Color(.black.withAlphaComponent(Constants.cardAmbientAlphaColor)),
```

Comment:
If possible it'd be nice to not have to go through UIColor here. This line could instead be 
```swift
                        .shadow(color: .black.opacity(Constants.cardAmbientAlphaColor),
```